docs/jdbc-mysql-catalog.md

```diff
@@ -100,9 +100,25 @@ Please refer to [Manage Metadata Using Gravitino](./manage-metadata-using-gravit
 MySQL doesn't support Gravitino `Boolean` `Fixed` `Struct` `List` `Map` `Timestamp_tz` `IntervalDay` `IntervalYear` `Union` `UUID` type.
 :::
 
+#### Table column auto-increment
+
+- Supports setting auto-increment.
+:::notice
+MySQL setting an auto-increment column requires simultaneously setting a unique index; otherwise, an error will occur.
```

Comment:
Is this notice mentioned in the official MySQL documentation? If so, could you provide the reference link?

Reply:
Then should we put the link to the doc?